notebooks/image_models/solutions/deepconv_gan.ipynb

```diff
@@ -50,33 +50,6 @@
     "## Import TensorFlow and other libraries"
```

Comment:
<p>Explain what <code>from_logits=True</code> means and why we need to use that (i.e. the discriminator outputs logits here and not probabilities)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Make sense. I added. Thanks!</p>